content/vault/v1.20.x/content/docs/deploy/kubernetes/vso/csi/index.mdx

```diff
@@ -0,0 +1,57 @@
+---
+layout: docs
+page_title: Direct secrets
+description: >-
+  Mount secrets from Vault directly into application containers with the Vault
+  Secrets Operator CSI driver.
+---
+
+# Direct secrets
+
+@include 'alerts/enterprise-only.mdx'
+
+While the Vault Secrets Operator lets you store Vault secrets in clusters as native
+Kubernetes Secrets, you can also use the "direct secrets" feature to mount Vault secrets directly to containers as [volumes](https://kubernetes.io/docs/concepts/storage/volumes/).
+
+The direct secrets feature installs the Vault Secrets
+Operator [CSI](https://github.com/container-storage-interface/spec/blob/master/spec.md) driver, which hooks into the Pod volume lifecycle directly,
+bypassing Kubernetes Secrets.
```

Comment:
I wonder if we could replace the "bypass" part with something like "secret data from Vault is stored in ephemeral Kubernetes Volumes" or something to that effect?